tools/flang1/flang1exe/accpp.c

```diff
@@ -914,6 +928,23 @@ accpp(void)
     }
   }
 
+  if (flg.list_macros) {
+    acc_display_macros();
+
+    FREE(deftab);
+    FREE(hashrec);
+    /* FREE(inclstack); Do not free this, gbl.curr_file points to it */
+    FREE(_ifs);
+    FREE(linebuf);
+
+    if (incllist != NULL) {
```

Comment:
Nit: Why this check for incllist and not for others?

Reply:
@kiranchandramohan i did it the same  as it is done in the rest file.Please look at lines ~1253